addons/controls/src/manager.tsx

```diff
@@ -9,7 +9,9 @@ addons.register(ADDON_ID, (api: API) => {
   addons.addPanel(ADDON_ID, {
     title() {
       const rows = useArgTypes();
-      const controlsCount = Object.values(rows).filter((argType) => argType?.control).length;
+      const controlsCount = Object.values(rows).filter(
+        (argType) => argType?.control && !argType?.table?.disable
```

Comment:
Hey @shilman the solution would be to use `includeConditionalArg` from `@storybook/csf`, but that'd require having access to args and globals. To get globals I can use use the `useGlobals` hook, but when trying to use the `useArgs` hook it fails with `Uncaught TypeError: Cannot read properties of undefined (reading 'id')`. Any ideas?

```suggestion
        (argType) => argType?.control && !argType?.table?.disable && includeConditionalArg(argType, args, globals)
```

Reply:
@yannbf Sounds like a question for @tmeasday. I'm guessing there's some situation where `useArgs` failed because there is no current story set and you're hitting that?

Reply:
Sounds like it! Though it seems like there are a lot of moving parts to get the correct argTypes. Sounds like it'd be nice to get a hook or a option of a hook that returns things correctly, already considering the filters and whatnot

Reply:
@yannbf so `useArgTypes` works, but `useArgs` doesn't? That's surprising, no?

How can I take a look?